### DISCUSSION TITLE
Notes from the Laboratory: October 2022 (and September too)

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#189] [Comments]
# Notes from the Laboratory: October 2022 (and September too)

Another monthly report from the depths of the laboratory of kollitsch.den - October 2022 (and also September 2022, not that you missed it ;)

https://kollitsch.dev/blog/2022/notes-from-the-laboratory-october/

**Comment by brycewray** [comment]
Sorry to hear about the health issues; hope you’re better now, sir.

**Reply by davidsneighbour** [reply]
It's a process ;)
